common/src/api/external/http_pagination.rs

```diff
@@ -312,6 +312,19 @@ pub type PaginatedByNameOrId<Selector = ()> = PaginationParams<
 pub type PageSelectorByNameOrId<Selector = ()> =
     PageSelector<ScanByNameOrId<Selector>, NameOrId>;
 
+pub fn id_pagination<'a, Selector>(
```

Comment:
I'm using this in `http_entrypoints.rs` to paginate over "group members", which only have UUIDs, not names.